Neos.ContentGraph.DoctrineDbalAdapter/src/ContentRepositoryReadModelAdapter.php

```diff
@@ -55,11 +57,13 @@ public function findWorkspaceByName(WorkspaceName $workspaceName): ?Workspace
     {
         $workspaceByNameStatement = <<<SQL
             SELECT
-                name, baseWorkspaceName, currentContentStreamId, status
+                ws.name, ws.baseWorkspaceName, ws.currentContentStreamId, cs.sourceContentStreamVersion as expectedSourceVersion, scs.version as sourceVersion
```

Comment:
As discussed: Maybe we can simplify this to sth like:
```suggestion
                ws.name, ws.baseWorkspaceName, ws.currentContentStreamId, cs.sourceContentStreamVersion = scs.version as isUpToDate
```